lib/Registry/Config.php

```diff
@@ -29,8 +29,13 @@ class Config extends ArrayObject
      */
     final public const NAME_REGEX = '{^[0-9a-zA-Z_-]+$}';
 
+    /** @var string */
     private $name;
 
+    /**
+     * @param string $name
+     * @param mixed[] $config
```

Comment:
not important but i guess this is the same as `array<array-key,mixed>`, but it really should always be `array<string,mixed>` IIRC